cucumber/authenticators_oidc/features/authn_oidc_v2.feature

```diff
@@ -70,6 +70,17 @@ Feature: OIDC Authenticator V2 - Users can authenticate with OIDC authenticator
     Then user "alice" has been authorized by Conjur
     And The authentication response includes header "X-OIDC-Refresh-Token"
 
+  @smoke
+  Scenario: A valid refresh token to get Conjur access token and OIDC refresh token
+    Given I enable OIDC V2 refresh token flows for "keycloak2"
+    And I fetch a code for username "alice" and password "alice"
+    And I authenticate via OIDC V2 with code
```

Comment:
Isn't this step emulating the redirect from the OIDC provider back to Conjur containing the code as a query param? My understanding is that this would always be a GET request but maybe I'm mistaken

Reply:
I don't think so. They're similar, but:
1. The OIDC redirect doesn't contain all the information it needs - it needs to be supplemented with both `nonce` and `code_verifier` parameters.
2. Conjur is responding to the `code` with an access token - the requester should be the Conjur client.

In terms of the UI:
1. The UI server redirects to the OIDC provider
1. The OIDC provider redirects back to the UI server using a GET request with `code` and `state` query parameters
2. The UI server will evaluate the `state`, and send the `code`, `nonce`, and `code_verifier` values to Conjur in a POST request
3. Conjur exchanges the `code` with the OIDC provider, and responds to the UI server with an access token